app/models/group_assignment.rb

```diff
@@ -0,0 +1,6 @@
+class GroupAssignment < ApplicationRecord
+  self.table_name = :groups_users
```

Comment:
Actually on reflection I think this should remain. The association table is `groups_users` as those are the tables we're linking, but this doesn't make for a descriptive name that fits the domain language on the associative model. `GroupAssignment` accurately describes this I feel.